features/gezag-minderjarige/adoptie.feature

```diff
@@ -0,0 +1,253 @@
+      | 1     | P            | 0         | 0       | 000000012         | Gerda     | gisteren - 20 jaar |
+      En de persoon 'Bert' heeft de volgende rijen in tabel lo3_pl_persoon
+      | pl_id | persoon_type | stapel_nr | volg_nr | burger_service_nr | voor_naam | geboorte_datum     |
+      | 2     | P            | 0         | 0       | 000000036         | Bert      | gisteren - 17 jaar |
+      Als 'Bert' is geadopteerd door 'Gerda' als ouder '1' op datum 'vandaag'
+      Dan heeft de persoon 'Bert' de volgende rijen in tabel lo3_pl_persoon
+      | pl_id | persoon_type | stapel_nr | volg_nr | burger_service_nr | voor_naam | geboorte_datum     | akte_nr | familie_betrek_start_datum |
+      | 2     | P            | 0         | 1       | 000000036         | Bert      | gisteren - 17 jaar |         |                            |
+      | 2     | P            | 0         | 0       | 000000036         | Bert      | gisteren - 17 jaar | 1AQ0100 |                            |
+      | 2     | 1            | 0         | 0       | 000000012         | Gerda     | gisteren - 20 jaar |         | vandaag                    |
+      En heeft de persoon 'Gerda' de volgende rijen in tabel lo3_pl_persoon
+      | pl_id | persoon_type | stapel_nr | volg_nr | burger_service_nr | voor_naam | geboorte_datum     |
+      | 1     | P            | 0         | 0       | 000000012         | Gerda     | gisteren - 20 jaar |
+      | 1     | K            | 0         | 0       | 000000036         | Bert      | gisteren - 17 jaar |
+
+  Regel: 'Gezag vragen van een persoon' specificeren voor de consumer developer
+
+    De scenarios opgenomen bij de vorige regels beschrijven het gedrag van de BRP API (wat is de response van de API bij een gegeven context)
+    De scenarios beschrijven niet wat de consumer developer moet doen om bijv. het gezag van een persoon te vragen. Met andere woorden, hoe moet de request er uit zien om het gezag van een persoon te vragen.
+    Onderstaand scenario is een voorbeeld voor het specificeren van de request voor het vragen van de burgerservicenummer, ouders en gezag velden van de persoon met burgerservicenummer '000000012'  
+
+    Scenario: Vragen van de velden 'burgerservicenummer, ouders en gezag' van een persoon met zijn burgerservicenummer
+      Gegeven de url van de personen bevragen endpoint is 'http://localhost/api/haalcentraal/personen'
+      Als de velden 'burgerservicenummer,ouders,gezag' van de persoon met burgerservice '000000012' moet worden gevraagd
+      Dan ziet de request er als volgt uit
+      """
+      POST http://localhost/api/haalcentraal/personen
+      Accept: application/json
+      Content-Type: application/json
+
+      {
+        "type": "RaadpleegMetBurgerservicenummer",
+        "burgerservicenummer": [ "000000012" ],
+        "fields" : [ "burgerservicenummer", "ouders", "gezag" ]
+      }
+      """
```

Comment:
ja, nu weet een consumer developer PRECIES wat te doen. Maar ook hier zit implementatiedetail in...
Ik weet eerlijk gezegd niet of dit nodig is om voor al deze doelgroepen te specificeren (heb ik nooit gevraagd...).  
Heb ik het goed begrepen dat wij onszelf de vraag moeten stellen: wat moet je wel en niet beschrijven voor wie?

Reply:
klopt. Ook als je een boek schrijft geeft je aan voor wie het is geschreven

Reply:
dit is nu implementatie documentatie en de beschrijvende documentatie komt